moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -122,6 +122,45 @@ export const ProjectsListViewFiltersConf = {
         "string_is_not_null",
       ],
     },
+    {
+      name: "task_order_name",
+      label: "Task orders",
+      placeholder: "Task order",
+      type: "string",
+      operators: [
+        "string_contains_case_insensitive",
+        "string_begins_with_case_insensitive",
+        "string_ends_with_case_insensitive",
+        "string_is_null",
+        "string_is_not_null",
```

Comment:
I see that the view is wrapping up 1 or more task order names into a comma delimited list. The result of that *is* a string, so this should work fine, but I wonder if we might ever want to expand these operators to do special stuff that handles arrays of values aggregated by the postgres database. This isn't a request for a change here, just a place maker for the idea of expanding the comparison operators.